app/packages/state/src/recoil/schema.ts

```diff
@@ -318,6 +320,38 @@ export const labelPaths = selectorFamily<
     },
 });
 
+const convertToLabelValue = (sampleId, path) => (raw) => {
+  const labelId = raw._id;
+  const field = path.split(".").shift();
+  return { labelId, field, sampleId };
+};
+
+export const labelValues = selectorFamily<
+  string[],
```

Comment:
nit: Aren't these Ritchie's changes? Rebase `run-updates` from `develop` and rebase `saveLastUsedSortBrainKey` from `run-updates`? No pressure, though. Just makes it easy to review 😁